app/models/batch_operation.rb

```diff
@@ -58,6 +59,8 @@ def dossiers_safe_scope(dossier_ids = self.dossier_ids)
       query.state_en_construction
     when BatchOperation.operations.fetch(:accepter) then
       query.state_en_instruction
+    when BatchOperation.operations.fetch(:follow) then
+      query.without_followers
```

Comment:
pas sur de la pertinence / de l’intérêt de ce scope.

Reply:
```suggestion
      query.without_followers.en_cours
```
cf: https://github.com/demarches-simplifiees/demarches-simplifiees.fr/blob/3dbfd81d95f9ed6758b41309c71b0d64783caffb/app/models/dossier.rb#L410